topic_monitor/setup.py

```diff
@@ -0,0 +1,39 @@
+from setuptools import setup
+
+setup(
+    name='topic_monitor',
+    version='0.0.0',
+    packages=[],
+    py_modules=[
+        'launch_files.launch_depth_demo',
+        'launch_files.launch_durability_demo',
+        'launch_files.launch_fragmentation_demo',
+        'launch_files.launch_reliability_demo',
+        'scripts.data_publisher',
+        'scripts.topic_monitor',
```

Comment:
The "best" way would be a `console_script` `entry_point` since it generate native executables on Windows. That is what all our Python packages do. A fall back would be to just install the `scripts` (https://docs.python.org/2/distutils/setupscript.html#installing-scripts).